packages/@stimulus/core/src/class_map.ts

```diff
@@ -12,7 +12,12 @@ export class ClassMap {
   }
 
   get(name: string) {
-    return this.data.get(this.getDataKey(name))
+    return this.getAll(name)[0]
+  }
+
+  getAll(name: string) {
+    const tokenString = this.data.get(this.getDataKey(name)) || ""
+    return tokenString.trim().split(/\s+/).filter(content => content.length)
```

Comment:
Should this be refactored out to `string_helpers` as something like `splitTokens`? Or wait until three usages?

Reply:
I went ahead and pulled this out into a function `tokenize` in `string_helpers.ts`. I did not update similar code in the `token_list_observer` as it lives in a separate package and is unrelated to the new functionality in this PR.